### PR TITLE
Website sale float qty module

### PR DESCRIPTION
**Pull request by StefanWiselka-Solvti**
Implementing module: website_sale_float_qty
This module adds:

New field in Unit of Measure - float_qty:boolean
When Float Qty is set to True, then products assigned with such UoM,
will give possibility to order decimal quantities on website shop.

Additionally, fixed issue in website_sale_minimum_order_quantity module.
- FIX: Wrongly passing 'kwargs' into super().

### CODE REVIEW

**Comment by StefanWiselka-Solvti** [comment]
Hi @janikvonrotz, Website sale float qty module is ready and can be tested/reviewed.
I hope to get some feedback.

So far, I've spent 17,5h.
If you wish, I can include some Unit Tests.
Best
Stefan

**Comment by janikvonrotz** [comment]
@MriV13 Thank you very much. I will check asap.

File: website_sale_float_qty/data/uom_data.xml
Comment (janikvonrotz) [inline website_sale_float_qty/data/uom_data.xml:1]: Can you move this record to demo instead of data?

File: website_sale_float_qty/demo/product_demo.xml
Comment (janikvonrotz) [inline website_sale_float_qty/demo/product_demo.xml:1]: Maybe rename it do demo.xml and add the `product_uom_decimal_meter` record here.

**Comment by janikvonrotz** [comment]
@MriV13 I have submitted a review. Do you see my comments?

**Comment by StefanWiselka-Solvti** [comment]
@janikvonrotz 
Hi, I've updated branch.
1. Moved UoM to demo data.
2. Renamed demo file.

**Comment by janikvonrotz** [comment]
@MriV13 Thank you very much :100: 

I will now provision the module on the customers server and ask for validation.

**Comment by StefanWiselka-Solvti** [comment]
> 



> @MriV13 Thank you very much 💯
> 
> I will now provision the module on the customers server and ask for validation.

@janikvonrotz Hi
Do you have some feedback from customers?
Did the module meet their expectations?

**Comment by janikvonrotz** [comment]
@MriV13 Hi Stephan

I haven't heard back from the customers. But I assume it meets the expectations.

I think we can close this request for now.

**Pull request merged**